README.md

```diff
@@ -17,6 +17,13 @@ are used for managing group accounts.
 * [Issue tracker](https://github.com/shadow-maint/shadow/issues)
 * [Releases](https://github.com/shadow-maint/shadow/releases)
 
+## Code
+
+The main development branch is at 
+ [https://github.com/shadow-maint/shadow.git](https://github.com/shadow-maint/shadow)
+
+See STABLE.md for a list of supported stable branches.
```

Comment:
Add reference to STABLE file. Example: [STABLE.md](STABLE.md)

Reply:
I'd thought about it, but wasn't sure what to link to.  I guess just

```
https://github.com/shadow-maint/shadow/blob/master/STABLE.md
```
?